concepts/05 UI Components/DataGrid/20 Editing/45 Disable Editing/30 Row.md

```diff
@@ -0,0 +1,210 @@
+If you want to forbid editing in a specific row, take the following steps:
```

Comment:
When you give step-by-step instructions, it's better to use an ordered list.

Reply:
ok